### PR TITLE
Exception on mismatch header

### PR DESCRIPTION
**Pull request by colinbird**

### CODE REVIEW

File: src/Http/CurlHandler.php [hunk 1]
```diff
@@ -169,6 +169,10 @@ protected function execute(array $options = [])
 
             $headerSize = $session->getInfo(CURLINFO_HEADER_SIZE);
 
+            if (strlen($result) < $headerSize) {
+                throw new Exception\TransferException('Header size(' . $headerSize . ') does not match result: ');
```
Comment (slavcodev) [inline src/Http/CurlHandler.php]: Why the colon in the end of message?

Comment (colinbird) [inline src/Http/CurlHandler.php]: I was going to add the output of $session->getInfo(null);
what do you think?

Comment (slavcodev) [inline src/Http/CurlHandler.php]: Hmm, I do not think we should add it to SDK. Who want to debug, can override the class.

Comment (colinbird) [inline src/Http/CurlHandler.php]: Ok so I will just remove the colon.. look ok?

**Pull request closed without merging**